### PR TITLE
[1LP][RFR] Add BZ marker and update test_server_name patterns for filling

### PR DESCRIPTION
**Pull request by mshriver**
{{pytest: cfme/tests/configure/test_server_name.py }}

PRT RESULTS:
5.11 failure is addressed in wt.patternfly PR #98 and #9084  (which will include update to wt.patternfly)

### CODE REVIEW

File: cfme/tests/configure/test_server_name.py [hunk 1]
```diff
@@ -23,27 +24,29 @@ def test_server_name(request, appliance):
...
-    appliance.server.settings.update_basic_information({'appliance_name': new_server_name})
-    flash_message = (
-        'Configuration settings saved for {} Server "{} [{}]" in Zone "{}"'.format(
-            appliance.product_name,
-            appliance.server.name,
-            appliance.server.sid,
-            appliance.server.zone.name))
-
-    view.flash.assert_message(flash_message)
+        appliance.rename(old_server_name)
```
Comment (john-dupuy) [inline cfme/tests/configure/test_server_name.py]: @mshriver It appears `rename` is a method of an `Appliance` instance but not an `IPAppliance` instance. https://github.com/ManageIQ/integration_tests/blob/976184c0df2ecee9e1baac5132facac5dd8552ef/cfme/utils/appliance/__init__.py#L2745

Do we want this to be method on `IPAppliance`?

Comment (mshriver) [reply cfme/tests/configure/test_server_name.py]: Agreed, moved.

File: cfme/tests/configure/test_server_name.py [hunk 2]
```diff
@@ -23,27 +24,29 @@ def test_server_name(request, appliance):
...
+
+    new_server_name = "RENAME-TEST"
+    assert view.server.basic_information.appliance_name.fill(new_server_name)
```
[the remark this answers is not in the document]
Comment (mshriver) [reply cfme/tests/configure/test_server_name.py]: Tighter assertion of the actions taken in the test is pretty much always a positive thing in my book.

Here I validate that the form field was actually modified by simply asserting the fill() call.
Next I validate that the save button is enabled after the form is modified by asserting save.is_enabled.
Then I assert that the `appliance.server.name`, which is fetched from REST interface, matches the update I made.
Then I assert the UI flash message
Then I assert the about modal reflects the updated name.

File: cfme/tests/configure/test_server_name.py [hunk 3]
```diff
@@ -23,27 +24,29 @@ def test_server_name(request, appliance):
...
+    assert view.server.save.is_enabled
+    view.server.save.click()  # no boolean return
+    assert appliance.server.name == new_server_name
+
+    view.flash.assert_success_message(
+        'Configuration settings saved for {} Server "{} [{}]" in Zone "{}"'
```
Comment (valaparthvi) [inline cfme/tests/configure/test_server_name.py]: Optional: Can we not make format this using keys?

Comment (mshriver) [reply cfme/tests/configure/test_server_name.py]: I would gladly merge a PR that changes this :) (already merged by the time I saw this)

**Pull request merged**